frontend/src/views/Pages/RegisterSelectCommunity.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div id="register-select-community">
+    <b-container class="justify-content-center text-center">
+      <div class="mb-4">{{ $t('community.current-community') }}</div>
+
+      <div v-for="community in communitys.community" :key="community.id" class="pb-3">
+        <b-card
+          v-show="community.name === $store.state.community_name"
```

Comment:
Or have you instored $store.state.community_name for this scenario?

Reply:
no, as already written, the name and the other community data information will come from the ENV or CONFIG.